### PR TITLE
Add a servlet for top songs

### PR DESCRIPTION
**Pull request by sashapff**

### CODE REVIEW

File: src/main/java/com/google/alpollo/Song.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    private final String name;
+    private final String album;
+
+    public Song(String artist, String name, String album) {
+        this.artist = artist;
+        this.name = name;
+        this.album = album;
+    }
+
+    public String artist() {
```
Comment (jeinygroove) [inline src/main/java/com/google/alpollo/Song.java]: Getter/setter should be called like 'getArtist'/'setArtist' to make obvious what this method does.

File: src/main/java/com/google/alpollo/Song.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        return artist;
+    }
+
+    public String name() {
```
Comment (jeinygroove) [inline src/main/java/com/google/alpollo/Song.java]: Same note about getter name.

File: src/main/java/com/google/alpollo/Song.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        return name;
+    }
+
+    public String album() {
```
Comment (jeinygroove) [inline src/main/java/com/google/alpollo/Song.java]: Same note about getter name.

File: src/main/java/com/google/alpollo/TopSongsServlet.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import com.google.gson.Gson;
+
+"/top"
+
+public class TopSongsServlet extends HttpServlet {
+    private final Gson gson = new Gson();
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
+        response.setContentType("text/html;");
```
Comment (jeinygroove) [inline src/main/java/com/google/alpollo/TopSongsServlet.java]: For json text use "application/json" content type

File: src/main/java/com/google/alpollo/TopSongsServlet.java [hunk 5]
```diff
@@ -0,0 +1,25 @@
+package com.google.alpollo;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+
```
Comment (jeinygroove) [inline src/main/java/com/google/alpollo/TopSongsServlet.java]: nit: remove empty line between imports

**Comment by jeinygroove** [comment]
LGTM

**Review by jeinygroove — APPROVED** [review]

File: src/main/java/com/google/alpollo/TopSongsServlet.java [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+import com.google.gson.Gson;
+
+"/top"
```
Comment (amaurymedeiros) [inline src/main/java/com/google/alpollo/TopSongsServlet.java:12]: Does that compile?

File: src/main/java/com/google/alpollo/Song.java [hunk 7]
```diff
@@ -0,0 +1,25 @@
+package com.google.alpollo;
+
+public class Song {
```
Comment (amaurymedeiros) [inline src/main/java/com/google/alpollo/Song.java:3]: Can you please add documentation to this and the other classes added in this PR?

File: src/main/java/com/google/alpollo/SongDataBase.java [hunk 8]
```diff
@@ -0,0 +1,13 @@
+package com.google.alpollo;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SongDataBase {
+    public static List<Song> topSongs() {
+        List<Song> songs = new ArrayList<>();
+        songs.add(new Song("The Weeknd", "Blinding Lights", "After Hours"));
```
Comment (amaurymedeiros) [inline src/main/java/com/google/alpollo/SongDataBase.java:9]: Please add an inline comment saying something like "we manually populate the top songs for now. TODO: automate how we populate the database"

**Pull request merged**